src/openms/include/OpenMS/ANALYSIS/ID/IDScoreSwitcherAlgorithm.h

```diff
@@ -260,9 +260,9 @@ namespace OpenMS
       }
       new_score_ = t;
 
-      if (type != ScoreType::RAW && higher_better_ != type_to_better_[type])
+      if (higher_better_ != type_to_better_[type])
```

Comment:
we currently enforce that all raw scores are higher better so I fix this here as well